ffi.md

```diff
@@ -4,6 +4,109 @@ title: Foreign Function Interfaces
 
 The ANSI Common Lisp standard doesn't mention this topic. So almost everything that can be said here depends on your OS and your implementation.
 
+### Example: Calling mathematical functions from C's `math.h` header file.
+ 
+This tutorial will use Guix to install cffi in an isolated shell environment. The REPL session below was tested in a sly repl.
```

Comment:
As a reader, I wonder if guix is necessary, why, and if I wanted to do without what would it take or what would be the limitations (in a few words).

Reply:
Guix is definitely not necessary here. I can remove that and send an update. I just made a quick draft of this.

I will say, that using Guix introduces a different workflow than using quicklisp, for example, because libraries can not be dynamically installed, updated, reloaded in a REPL without restarting emacs (there's a workaround).